cmd/cape/cmd/deploy.go

```diff
@@ -206,6 +206,7 @@ func doDeploy(url string, functionInput string, functionName string, insecure bo
 		Reader:    reader,
 		Insecure:  insecure,
 		PcrSlice:  pcrSlice,
+		Public:    public,
```

Comment:
Would this make a breaking change for creating public functions then?

Reply:
Currently public functions are only denoted by the lack of functionTokenPublicKey sent to runtime as part of the deploy request, this [change in runtime](https://github.com/capeprivacy/runtime/pull/333/files#diff-bebdee8ffdf8f4599411a605db6c7eae6ec9557b63d37c611afef97fe0ca2778R155) should gracefully be backwards compatible with the removal of function tokens for anyone using an old sdk/cli.

Reply:
Got it. Do these need to be merged in a certain order then?